examples/gno.land/p/demo/avl/z_0_filetest.gno

```diff
@@ -24,25 +24,18 @@ func main() {
 
 // Realm:
 // switchrealm["gno.land/r/test"]
-// u[a8ada09dee16d791fd406d629fe29bb0ed084a30:4]={
-//     "ObjectInfo": {
-//         "ID": "a8ada09dee16d791fd406d629fe29bb0ed084a30:4",
-//         "ModTime": "7",
-//         "OwnerID": "a8ada09dee16d791fd406d629fe29bb0ed084a30:7",
-//         "RefCount": "1"
-//     },
-//     "Value": {
-//         "T": {
-//             "@type": "/gno.RefType",
-//             "ID": "gno.land/p/demo/avl.Node"
-//         },
-//         "V": {
-//             "@type": "/gno.RefValue",
-//             "Hash": "627e8e517e7ae5db0f3b753e2a32b607989198b6",
-//             "ObjectID": "a8ada09dee16d791fd406d629fe29bb0ed084a30:5"
-//         }
-//     }
-// }
+// u[a8ada09dee16d791fd406d629fe29bb0ed084a30:4]=
+//     @@ -1,8 +1,8 @@
+//      {
+//          "ObjectInfo": {
+//              "ID": "a8ada09dee16d791fd406d629fe29bb0ed084a30:4",
+//     -        "ModTime": "0",
+//     -        "OwnerID": "a8ada09dee16d791fd406d629fe29bb0ed084a30:2",
+//     +        "ModTime": "7",
+//     +        "OwnerID": "a8ada09dee16d791fd406d629fe29bb0ed084a30:7",
+//              "RefCount": "1"
+//          },
+//          "Value": {
```

Comment:
this line and the next ones look weird, nope?

Reply:
I wanted to provide some context so that there is some idea of what is actually being modified; imo it is intelligible given the indentation of the diff. Do you think it should be done differently?

Reply:
same feeling.

Reply:
What are you folks proposing?

The "smart" ideal solution is one that can compute specific additions between one JSON and the other; like:

```
.ObjectInfo.ModTime = 0 -> 7
.ObjectInfo.OwnerID = "a8ada09dee16d791fd406d629fe29bb0ed084a30:2" -> "a8ada09dee16d791fd406d629fe29bb0ed084a30:7"
```

But it seems complex to try to achieve it and possibly overengineered.

Reply:
yes agree, it's overengineered. 
for me the discomfort comes from line 38, it's incomplete, otherwise it's good. thinking...

Reply:
so it's good to me.
Aside from that, only showing the diff seems to hide some other necessary information, so the ideal situation would be to dynamically adjust the context lines based on the length of the characters. However, that can be an optimization for the future.